e2e/cypress/integration/enterprise/guest_accounts/guest_experience_ui_spec.js

```diff
@@ -21,6 +21,9 @@ describe('Guest Account - Guest User Experience', () => {
             GuestAccountsSettings: {
                 Enable: true,
             },
+            ServiceSettings: {
+                EnableEmailInvitations: true,
```

Comment:
Is this really necessary?  If not, I suggest not to set.  Same case to other instances.

Reply:
Hi Saturn, yes this setting is required for Guest account invitations to work.

Reply:
Hmm, I set it to false and it still passed.

Reply:
Actually in this test since we are using the API to create the Guest user, it worked. But in other tests, we specifically invite users via email. Those will fail, if we set it to false.